stock_move_quick_lot/models/stock_move.py

```diff
@@ -13,12 +13,14 @@ class StockMove(models.Model):
         compute='_compute_line_lot_name',
         inverse='_inverse_line_lot_name',
     )
+    # Make store to avoid recompute it in onchange.
     life_date = fields.Datetime(
         string='End of Life Date',
         help='This is the date on which the goods with this Serial Number may '
              'become dangerous and must not be consumed.',
         compute='_compute_life_date',
         inverse='_inverse_life_date',
+        store=True,
```

Comment:
@sergio-teruel I suppose that the computation is slow in performances. Maybe the compute method should have been changed to make one search instead of one by line.